gate-web/src/test/groovy/com/netflix/spinnaker/gate/security/ldap/LdapAuthSpec.groovy

```diff
@@ -48,7 +48,7 @@ import static org.springframework.security.test.web.servlet.request.SecurityMock
 @Slf4j
 @GateSystemTest
 @SpringBootTest(
-    properties = ['retrofit.enabled=true','ldap.enabled=true', 'spring.application.name=gate', 'fiat.enabled=false', 'services.fiat.baseUrl=https://localhost'])
+    properties = ['retrofit2.enabled=true','ldap.enabled=true', 'spring.application.name=gate', 'fiat.enabled=false', 'services.fiat.baseUrl=https://localhost'])
```

Comment:
Do we need/want to set `retrofit.enabled=false` so we don't get a [RetrofitServiceFactoryAutoConfiguration](https://github.com/spinnaker/kork/blob/4e1a7974c001b5983d44a24d1b673857fb13f7ac/kork-retrofit/src/main/java/com/netflix/spinnaker/kork/retrofit/RetrofitServiceFactoryAutoConfiguration.java#L31) bean?